frontend/packages/kubevirt-plugin/src/components/vm-templates/instantiate-template/TemplateInfo.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { List, ListItem } from '@patternfly/react-core';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { ExternalLink } from '@console/internal/components/utils';
+import { TemplateKind } from '@console/internal/module/k8s';
+import { ANNOTATIONS } from '@console/shared';
+import { getTemplateOSIcon } from '../os-icons';
+import { TemplateResourceDetails } from './TemplateResourceDetails';
+
+export type TemplateInfoProps = {
+  template: TemplateKind;
+};
+
+export const TemplateInfo: React.FC<TemplateInfoProps> = ({ template }) => {
+  const { t } = useTranslation();
+  const { annotations, name } = template.metadata;
+  const { description } = annotations;
+  const displayName = annotations[ANNOTATIONS.displayName] || name;
+  const documentationURL = annotations[ANNOTATIONS.documentationURL];
+  const supportURL = annotations[ANNOTATIONS.supportURL];
+  const tags = (annotations.tags || '').split(',');
+  const osIconURL = getTemplateOSIcon(template);
+
+  return (
+    <div className="co-catalog-item-info">
+      <div className="co-catalog-item-details">
+        <span className="co-catalog-item-icon">
+          {osIconURL && (
+            <img
+              className="co-catalog-item-icon__img co-catalog-item-icon__img--large"
+              src={osIconURL}
+              alt=""
+            />
+          )}
+        </span>
+        <div>
+          <h2 className="co-section-heading co-catalog-item-details__name">{displayName}</h2>
+          {!_.isEmpty(tags) && (
+            <p className="co-catalog-item-details__tags">
+              {_.map(tags, (tag, i) => (
+                <span className="co-catalog-item-details__tag" key={i}>
```

Comment:
try to avoid `co-catalog...` css clases

Reply:
?

Reply:
removed `co-catalog...` CSS classes
added `template-form.scss` and `kv-template-form...` CSS classes to adjust UI as necessary